barretenberg/cpp/src/barretenberg/commitment_schemes/pairing_points.hpp

```diff
@@ -30,6 +29,8 @@ class PairingPoints {
     using VerifierCK = VerifierCommitmentKey<curve::BN254>;
 
   public:
+    static constexpr size_t PUBLIC_INPUTS_SIZE = 2 * Point::PUBLIC_INPUTS_SIZE;
```

Comment:
As discussed, would be ideal to remove code duplication between native and in-circuit of PUBLIC_INPUT_SIZE